subjects/java/projects/sharpen-it/audit/README.md

```diff
@@ -0,0 +1,59 @@
+#### Functional
+
+##### Check the installed plugins
+
+###### Can you find Junit and Nodejs or equivalents?
+
+##### Download the project and trigger a Jenkins build. Observe if the pipeline runs as expected.
+
+###### Does the pipeline initiate and run successfully from start to finish?
+
+##### Trigger some intentional build errors and observe Jenkins' response.
+
+###### Does Jenkins respond appropriately to build errors?
+
+##### Examine the automated testing step.
+
+###### Are tests run automatically during the pipeline execution? Does the pipeline halt on test failure?
+
+##### Make a minor change in the source code, create a new PR and observe if the pipeline is triggered automatically.
+
+###### Does a new PR automatically trigger the Jenkins pipeline?
+
+##### Check the deployment process.
+
+###### Is the application deployed automatically after a successful build?
+
+###### Is the rollback strategy well configured?
+
+##### Access the SonarQube web interface running on your local environment.
+
+###### Is the SonarQube web interface accessible?
+
+##### Integrate SonarQube with your GitHub repository.
+
+###### Is SonarQube integrated with GitHub, and does it trigger code analysis on every PR In the repository?
+
+##### Set up and configure SonarQube for code analysis using Docker.
+
+###### Is SonarQube configured correctly, and does it analyze code before the CI/CD pipeline?
+
+#### Comprehension
+
+##### Jenkins Setup and Integration
+
+###### Can the student explain what is the CI/CD and why we use it?
+
+###### Can the student explain every piece of the `Jenkinsfile`?
+
+###### Can the student describe the process of integrating Jenkins with the GitHub repository?
+
+##### SonarQube Setup and Integration
+
+###### Can the student explain the steps required to set up SonarQube within the project environment?
+
+###### Can the student describe the process of integrating SonarQube with the GitHub repository?
```

Comment:
Same here, only GitHub is mentioned: should we make it more "open" to GitLab, GitHub or any equivalent?